server/app/app.py

```diff
@@ -52,6 +52,22 @@ def run_scanpy(args):
         DATASET_TITLE=title,
         CXG_API_BASE=api_base
     )
+    app.features = {
```

Comment:
I think a `features()` method might be a good thing for `CXGDriver` classes to provide.

Reply:
✅